webui-src/app/mail/mail_resolver.js

```diff
@@ -0,0 +1,66 @@
+let m = require('mithril');
+let rs = require('rswebui');
+let util = require('mail_util');
+let widget = require('widgets');
+
+
+const Messages = {
+  all: [],
+  inbox: [],
+  sent: [],
+  outbox: [],
+  drafts: [],
+  load() {
+    rs.rsJsonApiRequest('/rsMsgs/getMessageSummaries', {},
+      (data) => {
+        Messages.all = data.msgList
+        // RS_MSG_BOXMASK = 0x000f
+        // RS_MSG_INBOX
+        Messages.inbox = Messages.all.filter(
+          (msg) => (msg.msgflags & 0x000f) === 0);
```

Comment:
Would be better to have constants with names that match the constants in rsmsgs.h instead of blind numbers (1,3,5, etc).